### DISCUSSION TITLE
Which UTC time slot should we choose for QoD meeting?

### DISCUSSION THREAD

**Discussion by RandyLevensalor** [camaraproject/QualityOnDemand#374] [General]
Propose a time based on UTC when we should have the QoD meeting.

The meeting is currently at 1200 UTC during CEST and would move to 1300 UTC during CET, if it were to continue to track Central European Time with daylight savings.

**Comment by RandyLevensalor** [comment]
1300 UTC

**Comment by benhepworth** [comment]
my vote is for 1300 UTC

**Comment by hdamker** [comment]
my vote as well for 1300 UTC

**Comment by eric-murray** [comment]
1300 UTC is fine for me too

**Comment by jlurien** [comment]
My vote is for 12:00 UTC

**Comment by hdamker** [comment]
13:00 UTC decided.
